mypy_django_plugin/lib/helpers.py

```diff
@@ -75,11 +76,11 @@ class DjangoTypeMetadata(TypedDict, total=False):
 
 
 def get_django_metadata(model_info: TypeInfo) -> DjangoTypeMetadata:
-    return cast(DjangoTypeMetadata, model_info.metadata.setdefault("django", {}))
+    return cast("DjangoTypeMetadata", model_info.metadata.setdefault("django", {}))
```

Comment:
Nit: Not opposed to this, but my first instinct is that these reduce readability.

From Ruff docs https://docs.astral.sh/ruff/rules/runtime-cast-value/#why-is-this-bad

> In some cases where `cast()` is used in a hot loop, this rule may also help avoid overhead from repeatedly evaluating complex type expressions at runtime.

I guess that's a point sometimes. But doesn't really apply when the type expression is a simple type reference.